tethys_cli/db_commands.py

```diff
@@ -0,0 +1,144 @@
+"""
+********************************************************************************
+* Name: db_commands.py
+* Author: Scott Christensen
+* Created On: 2019
+* Copyright: (c) Brigham Young University 2014
+* License: BSD 2-Clause
+********************************************************************************
+"""
+from pathlib import Path
+
+from django.conf import settings
+
+from tethys_cli.cli_helpers import get_manage_path, run_process, load_apps
+from tethys_apps.utilities import get_tethys_home_dir
+
+
+def add_db_parser(subparsers):
+    # DB COMMANDS
+    db_parser = subparsers.add_parser('db', help='Tethys Database Server utility commands.')
+
+    # Setup db commands
+    db_parser.add_argument('command',
+                           help='DB command to run.',
+                           choices=list(DB_COMMANDS.keys()))
+    db_parser.add_argument('-d', '--database', dest='db_alias',
+                           help="Name of the database options from settings.py to use.")
```

Comment:
I think I know what this is referring to, but maybe add an example like (e.g.: "DEFAULT").

Reply:
done